[c-6167] updating automations api docs with intro (#125)
trycourier/docs: versioned_docs/version-2.0.0/reference/automation/intro.mdx

---
sidebar_position: -1
sidebar_label: Introduction
pagination_label: Automations API
slug: ./
---

import Image from "@theme/IdealImage";

# Automations API

> Courier's Automations API let you trigger a series of steps —like send, delay, update profile, fetch-data— where each step represents a specific action that the automation will execute on your behalf when a single event is sent to the Automation API.

Reviewer: ```suggestion
Courier's Automations API lets you trigger a series of steps —like send, delay, update profile, and fetch-data. Each step represents a specific action that the automation will execute on your behalf. Automations are triggered when an event is sent to the Automation API.
```
